docs/guides/high-availability/concepts-multi-site.adoc

```diff
@@ -60,42 +56,41 @@ Monitoring is necessary to detect degraded setups.
 | Less than one minute
 
 | {jdgserver_name} cluster failure
-| If the {jdgserver_name} cluster fails in the active site, {project_name} will not be able to communicate with the external {jdgserver_name}, and the {project_name} service will be unavailable.
-The loadbalancer will detect the situation as `/lb-check` returns an error, and will fail over to the other site.
+| If the {jdgserver_name} cluster fails in one of the sites, {project_name} will not be able to communicate with the external {jdgserver_name} on that site, and the {project_name} service will be unavailable.
+The loadbalancer will detect the situation as `/lb-check` returns an error, and will direct all traffic to the other site.
 
-The setup is degraded until the {jdgserver_name} cluster is restored and the session data is re-synchronized to the primary.
+// TODO currently the failed site will automatically resume with out-of-sync state
+The setup is degraded until the {jdgserver_name} cluster is restored and the session data is re-synchronized.
 | No data loss^3^
 | Seconds to minutes (depending on load balancer setup)
 
 | Connectivity {jdgserver_name}
 | If the connectivity between the two sites is lost, session information cannot be sent to the other site.
-Incoming requests might receive an error message or are delayed for some seconds.
-The primary site marks the secondary site offline, and will stop sending data to the secondary.
-The setup is degraded until the connection is restored and the session data is re-synchronized to the secondary site.
+Incoming requests might receive an error message or are delayed for some seconds. One of the sites will automatically
+be taken offline by the stonith Lambda and will remain offline until the connection is restored and the session data is re-synchronized between the two sites.
```

Comment:
"stonith" -> "STONITH" as it is an acronym. On the same note, we might want to avoid the word the acronym due to the "shoot" that is part of the word. Maybe call it "fencing", which is still a war / martial arts term.

Reply:
`node eviction` is also an appropriate term here.

Reply:
I'll go with fencing.